website/static/js/pages/login-page.js

```diff
@@ -1,6 +1,25 @@
 /**
  * Login page
  */
-var LogInForm = require('js/signIn');
+var $ = require('jquery');
 
+var LogInForm = require('js/signIn');
 new LogInForm.SignIn('#logInForm');
+
+var SignUp = require('../signUp.js');
```

Comment:
Please move all `require` statements to the top of the module (before other statements, such as L7).